src/components/avatar/avatar.story.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs, text, boolean, select } from '@storybook/addon-knobs';
+import withReadme from 'storybook-readme/with-readme';
+import Section from '../../../.storybook/decorators/section';
+import Avatar from './avatar';
+import Readme from './README.md';
+
+storiesOf('Images', module)
```

Comment:
Not sure if this is the best path 🤔

Reply:
Let's put it top-level if there is no obvious path for now. We will group the components in a later step.